package.json

```diff
@@ -5,8 +5,8 @@
   "author": "QlikTech International AB",
   "license": "MIT",
   "scripts": {
-    "build": "webpack",
-    "build-watch": "webpack --config ./webpack.config-dev.js --watch",
+    "build": "concurrently \"webpack\" \"npm run test\" \"npm run lint\" \"npm run jsdoc\"",
```

Comment:
What's the benefit of running all commands here? It looks more like the commands you'd like to run as a pre-commit githook.